gen/nvml/device.go

```diff
@@ -38,21 +38,21 @@ func DeviceGetHandleByIndex(Index int) (Device, Return) {
 // nvml.DeviceGetHandleBySerial()
 func DeviceGetHandleBySerial(Serial string) (Device, Return) {
 	var Device Device
-	ret := nvmlDeviceGetHandleBySerial(Serial, &Device)
```

Comment:
My one question would be whether we should have a "helper" in `cgo_helpers` that wraps this? This would then be equivalent to the use of `clen`.

Not a blocker though.

Reply:
Let's consider this if we ever have to make an adjustment again.